src/components/menu/SubmenuItem.scss

```diff
@@ -10,15 +10,16 @@
         top: $bdl-submenu-offset;
         right: auto;
         left: calc(100% - #{$bdl-submenu-offset});
+        z-index: 9999;
```

Comment:
It seems like this conflicts with the other z-index changes we've made.

Reply:
Set to 171 which is one above menu z-index (170)